### PR TITLE
Seat Cover PDP improvements

### PR DESCRIPTION
**Pull request by coverlandjeff**
https://app.clickup.com/t/8688jym5e

### CODE REVIEW

**Review by george-icarcover — COMMENTED** [review]
Looks good to me. Just clean up code and await wills review as well

File: src/app/(main)/seat-covers/components/SeatCoverColorSelector.tsx [hunk 1]
```diff
@@ -1,58 +1,115 @@
...
+  },
+  {
+    display_color:"beige",
+    product:'http://www.coverland.com/custom-leather-seat-cover/01-seatcover-pc-be-1to.webp,http://www.coverland.com/custom-leather-seat-cover/02-seatcover-pc-be-1to.webp,http://www.coverland.com/custom-leather-seat-cover/03-seatcover-pc-be-1to.webp,http://www.coverland.com/custom-leather-seat-cover/04-seatcover-pc-be-1to.webp,http://www.coverland.com/custom-leather-seat-cover/05-seatcover-pc-be-1to.webp,http://www.coverland.com/custom-leather-seat-cover/06-seatcover-pc-be-1to.webp,http://www.coverland.com/custom-leather-seat-cover/07-seatcover-pc-be-1to.webp,http://www.coverland.com/custom-leather-seat-cover/08-seatcover-pc-be-1to.webp,http://www.coverland.com/custom-leather-seat-cover/09-seatcover-pc-be-1to.webp'
+ 
+  }
+]
```
Comment (devwilliamy) [inline src/app/(main)/seat-covers/components/SeatCoverColorSelector.tsx:47]: This needs to be in some database in the future

**Comment by devwilliamy** [comment]
Overall needs styles to be fixed

File: src/app/(main)/seat-covers/components/SeatCoverSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            {allOutOfStock(fullSeatCovers) ? <span className=""> - Out of Stock</span> : null}
+          </span>
+        ) : (
+          <span>Driver +  Passenger seats  {allOutOfStock(frontSeatCovers) ? <span className=""> - Out of Stock</span> : null}</span>
+        )}
+      </span>
+    </h3>
+    </div>
+  );
+};
```
Comment (devwilliamy) [inline src/app/(main)/seat-covers/components/SeatCoverSelector.tsx]: These should be extracted

File: src/app/(main)/seat-covers/components/SeatCoverSelector.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+          onClick={() => handleClick('full')}
+          className={`${buttonStyle} ${isSelectedNow === 'full' ? '!border-2	!border-solid	 border-gray-400 font-bold' : ''} ${allOutOfStock(fullSeat) ? ' cross font-normal :hover:bg-transparent	' : ''}`}
+        >
+          Full Seat
+        </Button>
+        {allOutOfStock(fullSeat) ? (<div className={`ofs-overlay-seat !h-[1%] !w-[52%] z-10 `}></div>) : null}
+      </div>
+    </>
+  );
+};
```
Comment (devwilliamy) [inline src/app/(main)/seat-covers/components/SeatCoverSelector.tsx]: These should be extracted

**Pull request merged**